node.js_mongodb/SingleUpload.js

```diff
@@ -0,0 +1,24 @@
+const DBHelper=require('./DB_helper');
+const readline = require('readline');
+PromptAndUpload();
+
```

Comment:
Also, let's add some docstring to the functions i.e.
/*
* Adds a new collection if it does not exist. If collection exists, replaces it with the given collection
*
*/